### PR TITLE
alumni account reset documentation

### PR DESCRIPTION
**Pull request by tahabi**
steps to take when alumni need access to their disabled accounts. Anything I got wrong or topics I may have missed?

### CODE REVIEW

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+These are the relevant steps to take in various situations.
+
+#### Missing CalNet UID    {calnet}
+
+If an alumni cannot use the online password reset function to reset the
+password on their account, but they do have a CalNet login, you can manually
+add the Calnet UID to their account in LDAP to let them perform a password
+reset. They may also need to have a Kerberos principal added for them.
+
+Please make sure to confirm the user's CalNet ID using the Berkeley Directory.
```
Comment (kpengboy) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: Alumni are often not listed in the Berkeley Directory, even if they have a CalNet login. I remember @daradib was able to confirm some identities via cal.berkeley.edu, maybe he can say more on that?

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+#### Disabled Account
+
+For various reasons, we've had to disable alumni accounts in the past. Some for weak
+passwords, some for security vulnerabilities, or myriad other reasons. If an account
+is [[sorried|doc staff/scripts/sorry]], once you've made sure the original problem
+has been resolved (you can check `User_Info` via [[check| doc staff/scripts/check]]
+you can [[unsorry|doc staff/scripts/unsorry]] their account and let them log in
+again. It may also be necessary to run `nscd -i groups` on **`tsunami`**, as well as
+`chown -R <username>:ocf ~<username>` to let them log in and access their files.
```
Comment (kpengboy) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: Don't forget their web dir

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md (hunk 1 above)
Comment (daradib) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: If you [search in the cal alumni network](https://cal.berkeley.edu/search/), alumni profile URLs are of the form `https://cal.berkeley.edu/profile.php?u=CALNETUID`.

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+[[!meta title="Alumni Account Reset"]]
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: we don't normally title case

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+If an alumni cannot use the online password reset function to reset the
+password on their account, but they do have a CalNet login, you can manually
+add the Calnet UID to their account in LDAP to let them perform a password
+reset. They may also need to have a Kerberos principal added for them.
+
+Please make sure to confirm the user's CalNet ID using the Berkeley Directory.
+
+To perform the association, simply follow the steps outlined in the
+[[LDAP Association|doc staff/procedures/accounts/association]] documentation
+with regards to adding the `calnetUid` record.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: but don't delete it afterwards

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+to manually add one.
+
+This error manifests itself as the following when a user attempts to reset
+their password:
+
+    kadmin Error: kadmin: cpw <username>: Principal does not exist
+
+To add the principal, run the following:
+
+    $ kadmin add <username>
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: i suggest `kadmin add --use-defaults --random-password <username>`

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+when kadmin presents them.
+
+#### Disabled Account
+
+For various reasons, we've had to disable alumni accounts in the past. Some for weak
+passwords, some for security vulnerabilities, or myriad other reasons. If an account
+is [[sorried|doc staff/scripts/sorry]], once you've made sure the original problem
+has been resolved (you can check `User_Info` via [[check| doc staff/scripts/check]]
+you can [[unsorry|doc staff/scripts/unsorry]] their account and let them log in
+again. It may also be necessary to run `nscd -i groups` on **`tsunami`**, as well as
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: either change to `passwd` (preferred) or correct the typo (`groups` -> `group`)

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md (hunk 2 above)
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: i suggest deleting this line, this shouldn't be necessary ever and we're going to do a one-time fix for existing broken accounts.

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md (hunk 5 above)
Comment (daradib) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: +1

File: ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    kadmin Error: kadmin: cpw <username>: Principal does not exist
+
+To add the principal, run the following:
+
+    $ kadmin add <username>
+
+You will need an admin principal yourself to do this. Choose the default options
+when kadmin presents them.
+
+#### Disabled Account
```
Comment (daradib) [inline ocfweb/docs/docs/staff/procedures/accounts/alumni-reset.md]: Move disabled account section to `scripts/unsorry.md` and replace with something like "If an account is [[sorried|doc staff/scripts/sorry]], refer to the [[unsorry|doc staff/scripts/unsorry]] page."

This is good info to have there and it's not alumni-specific.

**Comment by tahabi** [comment]
Any other thoughts before merging?

**Pull request merged**